src/slice.cr

```diff
@@ -163,7 +163,11 @@ struct Slice(T)
   # slice2 = slice + 2
   # slice2 # => Slice[12, 13, 14]
   # ```
+  @[NoInline] 
```

Comment:
I think this would have to be applied above the doc comments, otherwise the method would lose the documentation.

Ref https://github.com/crystal-lang/crystal/issues/6449

Reply:
Not on regular methods https://github.com/crystal-lang/crystal/issues/6449#issuecomment-495419391 . Just double checked